### PR TITLE
android zoomable image

### PR DESCRIPTION
**Pull request by chrisnojima**
- [x] image ios
- [x] video ios
- [x] image android
- [x] video android

### CODE REVIEW

**Comment by chrisnojima** [comment]
@keybase/react-hackers bump

File: shared/chat/conversation/attachment-fullscreen/index.native.tsx [hunk 1]
```diff
@@ -69,11 +69,18 @@ class _Fullscreen extends React.Component<Props & Kb.OverlayParentProps, {loaded
   _setLoaded = () => this.setState({loaded: true})
   render() {
     return (
-      <Kb.SafeAreaViewTop style={styles.safeAreaTop}>
+      <Kb.Box2
+        direction="vertical"
+        style={{backgroundColor: Styles.globalColors.black}}
```
Comment (hyperobject) [inline shared/chat/conversation/attachment-fullscreen/index.native.tsx]: this should probably be `globalColors.blackOrWhite` so the background isn't retina-scorchingly bright

Comment (hyperobject) [reply shared/chat/conversation/attachment-fullscreen/index.native.tsx]: (on dark mode)

Comment (chrisnojima) [reply shared/chat/conversation/attachment-fullscreen/index.native.tsx]: 👍 this code was from pre-dark mode

Comment (chrisnojima) [reply shared/chat/conversation/attachment-fullscreen/index.native.tsx]: fixed, also note blackOrWhite means, black in light mode and white in dark mode. we want blackOrBlack

Comment (hyperobject) [reply shared/chat/conversation/attachment-fullscreen/index.native.tsx]: ohhhh that's right

**Pull request merged**